modules/custom_operations/user_ie_extensions/tokenizer/python/README.md

```diff
@@ -40,7 +40,7 @@ compiled_tokenzier = compile_model(ov_tokenizer)
 text_input = "Test string"
 
 hf_output = hf_tokenizer([text_input], return_tensors="np")
-ov_output = compiled_tokenzier(pack_strings([text_input]))
+ov_output = compiled_tokenzier([[text_input]])  # TODO: Remove the second pair of square brackets when Python API is ready
```

Comment:
@jiwaszki, left TODO for now with the hope you mange to provide a way to avoid this extra pair of brackets.

Reply:
To check with: https://github.com/openvinotoolkit/openvino/pull/21734

Reply:
Works for me!